controller/lib/core/controller.cpp

```diff
@@ -31,11 +33,30 @@ static DebugFloat dbg_blower_valve_ki("blower_valve_ki",
 static DebugFloat dbg_blower_valve_kd("blower_valve_kd",
                                       "Derivative gain for blower valve PID");
 
+// In an ideal world we'd fully close the exhale valve during inhale, so as not
+// to waste any oxygen.  In practice having the valve be somewhat open is
+// helpful for the following reasons.
+//
+//  - If the exhale valve is not open, then (modulo leakage) pressure can never
+//    go down.  This means that if our control overshoots, we can't recover.
+//
+//  - With the blower at full speed, there's some leakage through the inhale
+//    pinch valve even when it's fully closed.  This means that pressure may
```

Comment:
This is _probably_ tunable from the pinch valve lookup table that linearizes the pinch valves: on my setup, the pinch valves "fully closed" position is not at the valve's stop, and I believe that if it were, that leakage would get smaller (but can it be completely removed is yet to be seen).

Reply:
See also https://github.com/RespiraWorks/VentilatorSoftware/pull/529 and #531.